builder/azure/common/client/config_test.go

```diff
@@ -158,6 +165,30 @@ func Test_ClientConfig_DeviceLogin(t *testing.T) {
 	}
 }
 
+func Test_ClientConfig_AzureCli(t *testing.T) {
+	getEnvOrSkip(t, "AZURE_CLI_AUTH")
+	cfg := Config{
+		UseAzureCLIAuth:  true,
+		cloudEnvironment: getCloud(),
+		authType:         authTypeAzureCLI,
```

Comment:
@nywilken I wasn't able to get the test running without adding authType. Can't see it being used with device login. Do you think I'm missing something here or is it correct to add `authType`?